gammapy/utils/docs.py

```diff
@@ -152,14 +149,17 @@ def modif_nb_links(folder, url_docs, git_commit):
                 nb.metadata['nbsphinx'] = {'orphan': bool('true')}
                 nb.cells.insert(0, new_markdown_cell(strcell))
                 nbformat.write(nb, filepath)
-            with open(filepath) as f:
-                txt = f.read()
+
+            with Path(filepath) as f:
+                txt = f.read_text(encoding="utf-8")
```

Comment:
Note that `Path.read_text` already has a with block that ensures the file handle is closed:
https://github.com/python/cpython/blob/1672c2fbae6128ee4717e08c4e65a0ab99a02a02/Lib/pathlib.py#L1190
So you can save one line:
```
txt = Path('example.txt').read_text()
```
I'll just make this tiny change now.